packages/data-models/src/options/options.js

```diff
@@ -24,7 +24,7 @@ export default class Options {
 
     this.defaults = defaults
     this.domain = defaults.domain
-    this.version = defaults.version
+    this.version = defaults.version.toString()
```

Comment:
is there any place we reference version as an int (e.g. to compare 2 versions?)

Reply:
It is defined as Number inside: 
`feature-options-defaults.json`
`language-options-defaults.json`
`site-options.json`
`ui-options-defaults.json`

in components packages.

I wanted simply to fix and convert it to String inside options's jsons. But after that I decided to add ability to support both variants.

Reply:
ok